docs/source/quickstart_drilldown.rst

```diff
@@ -0,0 +1,135 @@
+:orphan:
+
+.. _quickstart_drilldown:
+
+Quickstart options and troubleshooting
+============================================================
+
+..
+    Eventually, these H2s will probably all be separate articles. For now, I'm 
+    avoiding that so as not to create a bunch of super-skinny pages. 
+
+.. _quickstart_drilldown_install:
+
+Customize and troubleshoot MLflow installation
+---------------------------------------------------
+
+Python library options
+**********************
+
+Rather than the default MLflow library, you can install the following variations:
+
+.. list-table::
+   :widths: 10 20 70
+   :header-rows: 1
+
+   * - Name
+     - ``pip install`` command
+     - Description
+   * - mlflow-skinny
+     - ``pip install mlflow-skinny``
+     - Lightweight MLflow package without SQL storage, server, UI, or data science dependencies.
+   * - mlflow[extras]
+     - ``pip install mlflow[extras]``
+     - MLflow package with all dependencies needed to run various MLflow flavors. These dependencies are listed in doc:`https://github.com/mlflow/mlflow/blob/master/requirements/extra-ml-requirements.txt`
```

Comment:
Did we mean for this to say doc:`https://github.com/mlflow/mlflow/blob/master/requirements/extra-ml-requirements.txt`?

I think it would be better to make this a link to the file and remove the text "doc:"

Reply:
lol. DONE